vcx/wrappers/node/src/api/issuer-credential.ts

```diff
@@ -67,7 +67,7 @@ export class IssuerCredential extends VCXBaseWithState<IIssuerCredentialData> {
                          credentialName, price }: IIssuerCredentialCreateData): Promise<IssuerCredential> {
     try {
       const attrsVCX: IIssuerCredentialVCXAttributes = Object.keys(attr)
-      .reduce((accum, attrKey) => ({ ...accum, [attrKey]: [attr[attrKey]] }), {})
+      .reduce((accum, attrKey) => ({ ...accum, [attrKey]: attr[attrKey] }), {})
```

Comment:
Why did you leave this mapping? It is absolutely useless.
`
> var attr = {key: "value"}
> const attrsVCX = Object.keys(attr).reduce((accum, attrKey) => ({ ...accum, [attrKey]: attr[attrKey] }), {})
> console.log(attrsVCX)
{ key: 'value' }
`

An interface `IIssuerCredentialVCXAttributes` should be changed also because it specifies the old format.

Reply:
I needed someone to review to confirm that code wasn't needed any more.  correction checked in.